src/parser.ts

```diff
@@ -20,7 +20,7 @@ export function indentationLevel(line: string): number {
     return line.search(/\S|$/);
 }
 
-function nextIndentationLevel(parseOutput: IParseOutput, lines: string[], tabSize: number): number {
+function nextIndentationLevel(parseOutput: IParseOutput, lines: string[], tabSize: number, dedentIfHanging?: boolean): number {
```

Comment:
Same here, I'd rather update all existing tests and make the new param non-optional.